test/common/router/config_impl_test.cc

```diff
@@ -4023,7 +4031,7 @@ TEST_F(RouteMatcherTest, TestInvalidCharactersInHostRewrites) {
   routes:
   - match: { prefix: "/foo" }
     route:
-      host_rewrite: "new_host\ndroptable"
+      host_rewrite_literal: "new_host\ndroptable"
```

Comment:
Any use of v2 now fails across the tree unless runtime is set. For unit tests that aren't exercising deprecated features, I've upgraded them to v3 to avoid having to do the `TestDeprecatedV2Api` dance. I've tried to be consistent in following this philosophy, but please do call out places that might be non-obvious, this is a big toily patch.

Reply:
I'm mostly trying to understand how this was v2 before and now it's v3. I don't see any other changes in this file that changed the proto version?

Reply:
It was previously boosting, although I think the intent was for it not to, will fix.